autoload/codi/load.vim

```diff
@@ -118,13 +118,16 @@ endfunction
 
 let s:codi_default_interpreters = {
       \ 'python': {
-          \ 'bin': ['env', 'PYTHONSTARTUP=', 'python'],
+          \ 'bin': has('win32') ? ['python']
+          \    : ['env', 'PYTHONSTARTUP=', 'python'],
           \ 'prompt': '^\(>>>\|\.\.\.\) ',
           \ 'rephrase': function('s:rp_py'),
+          \ 'quitcmd': 'exit()',
           \ },
       \ 'javascript': {
-          \ 'bin': ['node', '-e', 'require("repl").start({ignoreUndefined: true, useGlobal: true})'],
+          \ 'bin': ['node', '-e', 'require(''repl'').start({ignoreUndefined: true, useGlobal: true})'],
```

Comment:
Are the double quotes meant to be changed to two pairs of single quotes? I'm not sure if it still works.

Reply:
Thank you